packages/nextjs/app/builders/0xA35D2C518710D3f953Ce4F69CDDEAaFc0c6b156c/page.tsx

```diff
@@ -0,0 +1,123 @@
+"use client";
```

Comment:
We don't need this restriction since your page doesn't use any client-only mechanism.

Please remove this line so we can improve the app's performance by pre-rendering the page during the compile process.